src/components/Log/LogDateItem.tsx

```diff
@@ -0,0 +1,23 @@
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import DateRangeIcon from '@mui/icons-material/DateRange';
+import { Box, Typography } from '@mui/material';
+
+interface LogDateItemProps {
+  date: Date | string | number;
+}
+
+export const LogDateItem = ({ date }: LogDateItemProps) => {
+  const dateObject = date instanceof Date ? date : new Date(date);
+
+  const dateString = dateObject.toLocaleDateString('nb-NO', { year: 'numeric', month: '2-digit', day: '2-digit' });
+  const timeString = dateObject.toLocaleTimeString('nb-NO', { hour: '2-digit', minute: '2-digit' });
+
+  return (
+    <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center' }}>
+      <DateRangeIcon color="primary" fontSize="small" />
+      <Typography>{dateString},</Typography>
```

Comment:
Personlig ville jeg kanskje likt bedre å droppe komma og heller ha et litt større mellomrom mellom dato og tidspunkt enn det er mellom ikon og tekst. Men ser skissene også bruker komma, så lever fint med det om dere liker det bedre slik også! :D

Reply:
Her er vi inne på alignment igjen😬 Om mellomrommet mellom ikon og tekst blir redusert, er ikke teksten på linje lenger. Har ikke sterke meninger her heller, men tenker det ikke er behov for komma på dato da.

![Screenshot 2024-11-22 at 12 56 57](https://github.com/user-attachments/assets/8fa7db85-9d10-4b91-baf3-1a7d20f67496)

Reply:
Ah, det er triksa for at dato og navn skal komme på linje ja :D Ser sikkert rart ut om de ikke er på linje ja.
Kan ev. legge til en ekstra `ml` på ikonet til høyre da, sånn at det får litt mer avstand til venstresiden?

Ang. komma så burde det være enten komma på begge linjene eller ingen av dem da. (På siste bilde der er det komma etter navnet mitt, men ikke etter datoen over)

Reply:
Tror jeg fjerner komma på navn også, da ikonet skaper et naturlig skille mellom de to uansett imo.